app/src/main/java/com/example/coronadiagnosticapp/ui/activities/OxymeterActivity.java

```diff
@@ -45,6 +39,7 @@
     // Variables Initialization
     private static final String TAG = "HeartRateMonitor";
     private static final AtomicBoolean processing = new AtomicBoolean(false);
+    private static final int MIN_LIGHT_VALUE = 170; // lux units.
```

Comment:
How did you choose this value?

Reply:
just some basic tests I made. The researchers will need to choose a better value

Reply:
ok